js/scripts.js

```diff
@@ -1,9 +1,4 @@
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
-}
+let books = [];
```

Comment:
- [ ] We are allowed to use Arrays but only inside a class. Remember we are refactoring our project to use classes that will hold data and provide us with methods that will help us manipulate the data(_the books array in this case_). See the screenshot below for reference:

| Requirement |
| --- |
| ![useClasses](https://user-images.githubusercontent.com/45002654/133329962-fb5ca6f1-8e06-4fcd-ad2b-d3a9de942e4f.PNG) |

Consider refactoring this array to the Book class as a static property. For example:

```JavaScript
class Book {
  static books = [ ];
  
  // if you would like to access this static property, start with the name of the class as below
  Book.books.push({ title: this.title, author: this.author });
}
```

Alternatively, you can have two classes, one that defines a book object and another that acts as a BookStore. In this case, the BookStore class will be the one to have the methods for adding and removing books.

Reply:
Created a Book class and a Bookshelf class to solve this.